ios/Plugin/Plugin.swift

```diff
@@ -192,6 +192,21 @@ public class CapacitorGoogleMaps: CustomMapViewEvents {
             }
         }
     }
+    
+    
+    @objc func removeMap(_ call: CAPPluginCall) {
+        let mapId: String = call.getString("mapId", "")
+
+        DispatchQueue.main.async {
+            guard let customMapView = self.customWebView?.customMapViews[mapId] else {
+                call.reject("map not found")
+                return
+            }
+            
+            (customMapView).view.removeFromSuperview()
+            self.customWebView?.customMapViews.removeValue(forKey: mapId)
```

Comment:
Can you add `call.resolve()` to the end of this method, so that it is resolved correctly.

Reply:
resolved